akka-stream/src/main/scala/akka/stream/javadsl/Sink.scala

```diff
@@ -116,6 +117,12 @@ object Sink {
     new Sink(scaladsl.Sink.actorSubscriber(props))
 
   /**
+   * Creates a `Sink` which uses Akka-IO's unconnected UDP mode to emit each ByteString it receives to the given target address.
+   */
+  def simpleUdp(host: String, port: Int): Sink[ByteString, Unit] =
```

Comment:
hmm, I would go with the connected one in this case. The unconnected can be a Sink[(host, port, ByteString), Unit].